templates/organization/base-organization.html

```diff
@@ -23,43 +23,23 @@ <h1>
-                <a href="{% url 'edit-organization-personnel' organization.uuid first_personnel.id %}" class="btn btn-primary">{% trans "Personnel" %}</a>
-            {% elif first_personnel %}
-                <a href="{% url 'edit-organization-personnel' organization.uuid first_personnel.id %}" class="btn btn-default">{% trans "Personnel" %}</a>
-            {% elif organization %}
-                <a href="{% url 'create-organization-personnel' organization.uuid %}" class="btn btn-default">{% trans "Personnel" %}</a>
-            {% endif %}
-            {% if first_emplacement and current_emplacement or first_association and current_association %}
-                {% if current_emplacement %}
-                <a href="{% url 'edit-organization-emplacement' organization.uuid first_emplacement.id %}" class="btn btn-primary">{% trans "Locations" %}</a>
-                {% elif current_association %}
-                <a href="{% url 'edit-organization-association' organization.uuid first_association.id %}" class="btn btn-primary">{% trans "Locations" %}</a>
-                {% endif %}
-            {% elif first_emplacement or first_association %}
-                {% if first_emplacement %}
-                <a href="{% url 'edit-organization-emplacement' organization.uuid first_emplacement.id %}" class="btn btn-default">{% trans "Locations" %}</a>
-                {% else %}
-                <a href="{% url 'edit-organization-association' organization.uuid first_association.id %}" class="btn btn-default">{% trans "Locations" %}</a>
-                {% endif %}
-            {% elif organization %}
-                <a href="{% url 'create-organization-emplacement' organization.uuid %}" class="btn btn-default">{% trans "Add site" %}</a>
-                <a href="{% url 'create-organization-association' organization.uuid %}" class="btn btn-default">{% trans "Add area" %}</a>
+            {% if organization.uuid %}
+              <a href="{% url 'edit-organization' organization.uuid %}"
+                  class="btn btn-{% if is_basics_active %}primary{% else %}default{% endif %}">{% trans "Basics" %}</a>
+              <a href="{% url "create-organization-composition" organization.uuid %}"
+                  class="btn btn-{% if is_composition_active %}primary{% else %}default{% endif %}">{% trans "Relationships" %}</a>
+
+              <a href="{% url 'create-organization-personnel' organization.uuid %}"
+                  class="btn btn-{% if is_personnel_active %}primary{% else %}default{% endif %}">{% trans "Personnel" %}</a>
+
+              <a href="{% url 'create-organization-emplacement' organization.uuid %}"
+                  class="btn btn-{% if is_location_active %}primary{% else %}default{% endif %}">{% trans "Locations" %}</a>
+           {% else %}
+             <a href="{% url 'create-organization' %}"
+                 class="btn btn-{% if is_basics_active %}primary{% else %}default{% endif %}">{% trans "Basics" %}</a>
```

Comment:
What's the scenario where this `else` block is useful? Currently, the only effect I can see it having is that it renders a non-active `Basics` button when you're creating an organization:

<img width="986" alt="Screen Shot 2019-08-16 at 9 32 10 AM" src="https://user-images.githubusercontent.com/14170650/63175022-c4784b00-c008-11e9-8663-c13ec8991d25.png">

If that's the case, I think we can probably get away with making this whole widget conditional on `organization.uuid` existing, including the `Edit:` text and all butons.

Reply:
that's a good point—i'll remove the edit section altogether. the user is already anchored by "Add a new organization."